### ISSUE TITLE
Identity Labels

### ISSUE THREAD

**Issue by david1aa** [speedball2001/identitychooser-mx#26]
I have quite a few identities I use (over a hundred), and it gets difficult to find the one I need in the moment. If you have the add-on add the identity labels for each identity in the drop-down list, it would make this much easier. Alphabetizing by identity label would be a great addition, as well.

**Comment by speedball2001** [comment]
Please try the latest beta version https://github.com/speedball2001/identitychooser-mx/releases/tag/2.3beta1. It has identity label support.

As for ordering, you could re-order the identities in Identity Chooser's settings. You have to do it manually, though, which will be tedious if you have a hundred, but it's there as an option.

**Comment by david1aa** [comment]
Yes, the beta works much better. I did have an issue. The first time I installed it, I couldn't access the options, and it deinstalled when I closed the add-ons page. The second installation worked fine, and stayed put.

It's too bad the Manage Identities window doesn't have headers to sort them with. Perhaps I should make a suggestion to Mozilla to add that function. Sorting them through the add-on settings wasn't too difficult, but yes, tedious. I only have to do it once, though. I can easily sort any new ones I create from now on. Now, if I had a thousand of them... 

Thanks for making this add-on. The stock functionality definitely leaves something to be desired, and this add-on makes it much better for my purposes. I create a new identity every time I sign up for something. That way, if I get spammed, I know who the culprit is. I can deal with them accordingly. Now I can easily find the identity I'm looking for, when I need to respond to an email or website.
